.github/workflows/ci_docs.yml

```diff
@@ -137,7 +139,20 @@ jobs:
 
       # Build the documentation
       - name: Build the documentation
-        run: make -C doc clean all
+        run: make -C doc clean html
+
+      - name: Build the PDF
+        run: |
+          make -C doc pdf
+          ls -lh doc/_build/latex/*
+
+      # Temporarily upload the built PDF to the artifacts
+      - name: Upload PDF to artifacts
+        uses: actions/upload-artifact@v4.5.0
+        with:
+          name: pygmt-docs-pdf
+          path: doc/_build/latex/pygmt.pdf
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
Uploading the PDF to artifacts for debugging purpose. Should remove the step before merging.